Dockerfile

```diff
@@ -25,14 +25,12 @@ COPY version/ version/
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
-#FROM registry.access.redhat.com/ubi8/ubi-micro:latest
-WORKDIR /
-COPY --from=builder /workspace/manager .
-COPY --from=builder /workspace/resources/ ./resources/
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
we probably can evaluate the usage of the ubi8-micro image, do you mind testing it?

Reply:
ok

Reply:
The ubi8-micro is working well both for dev mode that for production mode, I think we can use that.
I did the update

Reply:
great, thanks @davidesalerno